doc/tutorial/ranges/views/minimiser/index.md

```diff
@@ -0,0 +1,161 @@
+# Minimiser {#tutorial_minimiser}
+
+[TOC]
+
+This tutorial introduces minimisers. Minimisers are a compact representation of a sequence that is closely related to,
+but more efficient than a representation by k-mers. The minimisers are implemented as a view in SeqAn3. For more
+information about views and how to implement your own view, please see  
+\ref tutorial_ranges and \ref howto_write_a_view .
+
+\tutorial_head{Easy, 20 min, , \ref tutorial_ranges\, \ref howto_write_a_view}
+
+# Motivation
+
+A common way to work with sequences is to obtain their k-mers, but even with a large size of k, the number of k-mers is
+non-trivial, therefore storing k-mers is costly. At the same time, due to the overlap of consecutive k-mers, the
+information they contain is highly redundant. That is why, minimisers come in handy. Minimisers are k-mers, which are
+minimal in a window of a specified size.  Minimal could for example mean lexicographically smallest. By storing only
+these minimal k-mers the storage cost is significantly reduced while maintaining a similar amount of information.
+
+# Minimiser Workflow
+
+Because minimisers are minimal k-mers, they depend on a given k-mer size, a given shape (specifying which positions
+should be considered) and a window size, which has to be greater or equal than the k-mer size. If all these values are
```

Comment:
```suggestion
should be considered) and a window size, which has to be greater than the k-mer size. If all these values are
```
I think?

Reply:
It can be also equal, then you get back the minimum between forward and backward strand. :)

Reply:
ah alright!